packages/bigtable/src/table.js

```diff
@@ -659,7 +677,17 @@ Table.prototype.getRows = function(options, callback) {
  *   }
  * ];
  *
- * table.insert(entries, function(err, apiResponse) {});
+ * table.insert(entries, callback);
+ *
+ * //-
+ * // A stream mode is also provided, this is useful when making a large number
```

Comment:
You may also insert entries to the table as a stream. This is useful when making a large number of inserts.